contracts/MechMarketplace.sol

```diff
@@ -307,14 +297,6 @@ contract MechMarketplace {
         if (priorityMech == address(0)) {
             revert ZeroAddress();
         }
-        // Agent mech itself cannot post a request
-        if (mapMechRegistrations[msg.sender]) {
-            revert UnauthorizedAccount(msg.sender);
-        }
```

Comment:
why was this dropped?

Reply:
Discussed, mech marketplace is going to be ownerless, and mech registration is not needed, as all the validity is based on the staking factory verification.